app/assets/stylesheets/1st_load_framework.css.scss

```diff
@@ -0,0 +1,92 @@
+// import the CSS framework
+@import "foundation";
+
+// override for the 'Home' navigation link
+.top-bar .name {
+  font-size: rem-calc(13);
+  line-height: 45px; }
+.top-bar .name a {
```

Comment:
Merge rule `.top-bar .name a` with rule on line 5<br>Selector should have depth of applicability no greater than 2, but was 3